cli/stream.py

```diff
@@ -343,18 +373,12 @@ def stream(
         entity_types = calculate_entity_value(entity_types)
         output_types = list(set(generate_output_types(entity_types)))
     else:
-        domain_dict = Domain.get_all_domain_dict()
-        parse_output_types = set()
-
-        for output_type in output_types.split(","):
-            output_type = to_snake_case(output_type)
-            if output_type not in domain_dict:
-                raise click.ClickException(f"Output type {output_type} is not supported")
-            parse_output_types.add(domain_dict[output_type])
+        output_types = generate_dataclass_type_list_from_parameter(output_types, "output")
 
-        if not output_types:
-            raise click.ClickException("No output types provided")
-        output_types = list(set(parse_output_types))
+    if source_types is None and source_path.startwith("postgresql://"):
+        source_types = "block,transaction,log"
+    if source_types:
```

Comment:
Should we add this extra step or we should enforce user to list all the source types?

Reply:
Instead of this step, I set a default value to “--source-types”